Project-Sloth/ps-realtor
# Added Finalize Confirm Button

# Overview
Adds a Confirm Button on the UI for selling.

# Details
This adds a simple confirm button when finalizing the property sale, this avoids the function spamming when entering IDs, It will only trigger the function when you press confirm.

# UI Changes / Functionality
![image](https://github.com/Project-Sloth/ps-realtor/assets/74205343/c0f5dc76-47e4-4c9c-a0e9-fea068422aaf)
![image](https://github.com/Project-Sloth/ps-realtor/assets/74205343/cf475fb8-9469-4623-8191-4635d8ef49dc)

# Testing Steps
- Open the UI
- Create a Property
- Sell a house
- You can also try to resell it to a different person, the confirm button should only show when there is input or when the new owner ID/citizen ID is different than the current

- [x] Did you test the core functionality of the script to ensure your changes do not regress other areas?
